src/components/menus/branch-menu.js

```diff
@@ -75,21 +75,32 @@ const withBranchMenu =
             );
```

Comment:
need to take hvdc-line into account

Reply:
the menu is empty for hvdc lines ?

Reply:
For now, we just need "view in spreadsheet" in hvdc lines menu